src/wrappers/cryptosignwrapper.cpp

```diff
@@ -64,6 +79,59 @@ bool verify(std::string const &signature, std::string const &message, bytes cons
     return res == 0;
 }
 
+bytes scrypt(std::string const &password, KdfParams const &kdfParams)
+{
+    unsigned int const keyLength = kdfParams.dklen;
+    unsigned char derivedKey[keyLength];
+
+    auto passw = reinterpret_cast<const unsigned char*>(password.data());
```

Comment:
Can we extract `reinterpret_cast<const unsigned char*>` to a utility function (or a macro?) called `as_char_ptr` or something like that? (used quite often in this file). Just a suggestion.

Reply:
I'm usually not a fan of macros, but here it comes in very handy. 
I've applied your suggestion for this whole file